### DISCUSSION TITLE
samsung S22 problem with handover using 2 benetel650 O-RU

### DISCUSSION THREAD

**Discussion by abcd11abcd** [srsran/srsRAN_Project#928] [General Help]
We are using samsung galaxy s22 for handover between 2 cells(each benetel o-ru with one cell) which are configured with same frequency. In this case the s22 mobile is not even recognizing a single cell. But if we run single cell setup with each ru separately, the mobile gets attached and everything seems fine. But when we configured 2 cells, then only this issue is happening. 
on the other hand, the samsung A25 mobile is working with 2 cell and handover is also happening. A25 is working perfectly fine with same frequency in both cells and for different frequency also it worked properly.
Any S22 users kindly help

**Comment by ujjwalpawar** [comment]
any update? Can you share your configuration files used?
